BEProjectsApp/models.py

```diff
@@ -80,3 +98,50 @@ def publish(self):
 
     def __str__(self):
         return self.title
+
+
+class OH_Project(models.Model):
```

Comment:
Can we give this model a better name? You can suggest any name you have in your mind.

Reply:
OutHouse_Project  ? or
how about Out_House_Porject

Reply:
Or how about `Outhouse_Project`?

Reply:
that sounds cool too i will implement that